ak.py

```diff
@@ -19,6 +19,45 @@
 MODULE_FOLDER = WORKSPACE + 'parts/'
 
 
+class DbTools(object):
+    """Read db credentials from ERP_CFG.
+
+    Add -d flag to the current command to override PGDATABASE
+    Add self.db
+
+    Usage:
+      Heritate from this class and call determine_db()
+
+        class AkSomething(cli.Application, DbTools):
+            def main(self):
+                self.determine_db()
+                # your stuff here
+    """
+
+    dbParams = {
+        "db_host": "PGHOST",
+        "db_port": "PGPORT",
+        "db_name": "PGDATABASE",
+        "db_user": "PGUSER",
+        "db_password": "PGPASSWORD"
+    }
+    db = cli.SwitchAttr(["d"], str, help="Database")
+
+    def determine_db(self):
```

Comment:
determine looks rather "frenchouillard", o at least it's not the right meaning in English I think. May be get_db or find_db...